vuepress/docs/next/tutorials/micro-frontends/react.md

```diff
@@ -126,106 +126,105 @@ import './WidgetElement';
 You’re now running `React` in a containerized micro frontend.
 :::
 
-## Get Resource URL
+## Get the Resource URL
 
-Let's add our micro frontend to Entando by adding our JavaScript and CSS files to the `public` folder.
+Add your micro frontend to Entando by uploading the JavaScript and CSS files to the `public` folder. This is the way Entando makes files available to the public.
 
-::: tip
-Entando makes files publicly available through the `public` folder.
-:::
+### Add Widget
 
-#### Add Widget
+First, add a widget to get the resource URL for the `public` folder. Then, we'll use the same widget to add our micro frontend to Entando.
 
-Initially, we'll add a widget to get the resource URL for the `public` folder. Later on, we'll use the same widget to add our micro frontend to Entando.
+1. Go to `Components > Micro frontends & Widgets` in the App Builder
 
-1. Go to `Components > Micro frontends & Widgets` in the Entando App Builder.
+2. Click `Add` at the lower right corner
 
-2. Click `Add` at the lower right.
-
-![New widget screen](./img/new-widget-screen.png)
+![New widget screen](./img/new-addwidget-screen.png)
```

Comment:
Hmm, did I miss this on the previous review? It's best to update the existing image with the same name so we don't wind up with unused images in the system which can confuse things later.

Reply:
done

Reply:
looks good